### ISSUE TITLE
No ARM Python on macOS possible

### ISSUE THREAD

**Issue by Fabian188** [spyder-ide/spyder#21504]
I have a M1 Mac, hence my local python installation is arm based via `/opt/homebrew`, the x86_64 code would be in `/usr/local`

It seems that Spyder on Mac comes with x86_64 anaconda, as 
```
import platform
platform.machine()
```
gives 'x86_64'.

If I want to change the python interpreter, I get the error 
```
Invalid file path:
/opt/homebrew/bin/ipython
```
However, this is the proper path but to an arm variant.

I assume that spyder itself is already started from a x86_64 python.

**Comment by ccordoba12** [comment]
Hey @Fabian188, thanks for reporting. You said

> If I want to change the python interpreter, I get the error
> Invalid file path:
> /opt/homebrew/bin/ipython

That's because you need to select a **Python** interpreter, not an IPython one.

In your case I guess that would be

    /opt/homebrew/bin/python

or maybe

    /opt/homebrew/bin/python3

if the first one is not present.

Let us know if that solves the problem for you.

**Comment by Fabian188** [comment]
Thank you, that helped!

I had to install spyder-kernels on my local python which is only available as 2.5.0 but with the current Spyder 5.5.0 it works!

I use as interpreter /opt/homebrew/bin/python3.11 which is arm64

Spyder itself runs with Python 3.9.14 64-bit | Qt 5.15.11 | PyQt5 5.15.10 | macOS 14.0 (x86_64) 
But the emulation on Mac is very fast, I doubt I would see a difference to Spyder itself as arm.

**Comment by ccordoba12** [comment]
Yeah, I doubt you'd see much of a difference with a native version.

And thanks for letting us know that you can use Spyder without problems now.
